src/commonMain/kotlin/com/benasher44/uuid/uuid.kt

```diff
@@ -9,26 +9,6 @@ internal const val UUID_STRING_LENGTH = 36
 @Deprecated("Use `Uuid` instead.", ReplaceWith("Uuid"))
 public typealias UUID = Uuid
 
-/**
- * Construct new [UUID] instance using the given data.
- *
- * @param msb The 64 most significant bits of the [UUID].
- * @param lsb The 64 least significant bits of the [UUID].
- */
-// @SinceKotlin("1.x")
-@Suppress("FunctionName")
-public fun Uuid(msb: Long, lsb: Long): Uuid =
```

Comment:
For improved compatibility, shouldn't we make this the constructor and make the one that takes the ByteArray a free function?

Reply:
My argument here is that we should make all freestanding and avoid any possible compatibility issues with anything. It's the most future proof decision and the most coherent design.

Reply:
I generally agree, but if we are going to have to have an expect/actual somewhere then: either the below function or the UUID class constructor

Reply:
This one would become the `expect` and the JVM definition would become `public actual inline fun uuidOf(msb: Long, lsb: Long): Uuid = java.util.UUID(msb, lsb)`. This way there’s no overhead on JVM for it, no?

Reply:
Ah makes sense. Thanks!